.changeset/neat-laws-matter.md

```diff
@@ -0,0 +1,5 @@
+---
+"@primer/css": patch
+---
+
+Use standard lifecyle milestones for statuses.
```

Comment:
Not everthing needs to be attached to a release, and I think since this is doc updates, it can go out without this file. 📃 🗑️